dockerized/default-env

```diff
@@ -1,4 +1,6 @@
 NODES=<comma separated ndoes>
 PORT=<port to expose>
 LOG_LEVEL=<log level>
-JWT_SECRET=<jwt secret>
\ No newline at end of file
+JWT_SECRET=<jwt secret>
+RELEASE_TAG=<specify release tag if desired stable release build>
+FCU_MAJORITY=0.6
```

Comment:
No need to specify the fcu majority flag here as it's 0.6 as it's set by default if not set.

Reply:
thanks for the suggestions, fixed on a new commit